src/plugins/github/plugins/publish/utils.py

```diff
@@ -33,26 +33,18 @@
         Issue,
         PullRequest,
         PullRequestSimple,
-        PullRequestSimplePropLabelsItems,
     )
 
 
-def get_type_by_labels(
-    labels: LabelsItems | list["PullRequestSimplePropLabelsItems"],
-) -> PublishType | None:
-    """通过标签获取类型"""
-    if not labels:
-        return None
-
+def get_type_by_labels(labels: PullRequestLabels) -> PublishType | None:
+    """通过拉取请求的标签获取发布类型"""
     for label in labels:
         if isinstance(label, str):
             continue
-        if label.name == PublishType.BOT.value:
-            return PublishType.BOT
-        if label.name == PublishType.PLUGIN.value:
-            return PublishType.PLUGIN
-        if label.name == PublishType.ADAPTER.value:
-            return PublishType.ADAPTER
+        for type in PublishType.members():
```

Comment:
```suggestion
        for type in PublishType:
```

如果要这种写法的话，为什么不直接这样呢？